lib/runner/extensions/event.command.js

```diff
@@ -92,6 +92,26 @@ sanitizeFiles = function (request, callback) {
     return callback(null, request, sanitisedFiles);
 };
 
+/**
+ * Return the item path
+ *
+ * @param {any} item - sdk item
+ */
+function getItemPath (item) {
+    let path = '',
+        current = item;
+
+    while (current) {
+        if (current.name) {
+            path = current.name + (path !== '' ? '/' : '') + path;
+        }
+
+        current = current.__parent;
+    }
+
+    return path;
```

Comment:
We can avoid the complex logic here:

```suggestion
function getItemPath (item) {
    const pathParts = [];
    let current = item;

    while (current) {
        if (current.name) {
            pathParts.push(current.name);
        }

        current = current.__parent;
    }

    return pathParts.reverse().join('/');
```